testing/unit_tests/t_utils.py

```diff
@@ -45,8 +45,9 @@ def create_tmp_dir(copy_data_testing_dir=True):
             into the ``tmp`` folder.
     """
     remove_tmp_dir()
-    Path(path_temp).mkdir()
+    Path(path_temp).mkdir(parents=False, exist_ok=True)
     if Path(path_data_testing_source).exists() and copy_data_testing_dir:
+        shutil.rmtree(path_data_testing_tmp, ignore_errors=True)
```

Comment:
note with python 3.8+, we can have exist_ok for copytree, so this line would not be needed

Reply:
Yup. we are still supporting 3.7 sadly. Definitely removable in the future.